changelog.d/10938.bugfix

```diff
@@ -0,0 +1 @@
+Fix [MSC2716](https://github.com/matrix-org/matrix-doc/pull/2716) `/batch_send` endpoint not being able to create outlier events.
```

Comment:
we should say which version the bug was introduced in:

```suggestion
Fix bug introduced in Synapse 1.44 which caused the experimental [MSC2716](https://github.com/matrix-org/matrix-doc/pull/2716) `/batch_send` endpoint to return a 500 error.
```

(can we be more specific? "create outlier events" is too opaque for an admin - what causes it to generate outliers?)